packages/@dcl/inspector/src/redux/data-layer/index.ts

```diff
@@ -30,24 +46,48 @@ export const dataLayer = createSlice({
     reconnect: (state) => {
       console.log('[WS] Reconnecting')
       state.error = ErrorType.Reconnecting
-      state.dataLayer = undefined
+      dataLayerInterface = undefined
     },
-    connected: (state, { payload }: PayloadAction<{ dataLayer: DataLayerState['dataLayer'] }>) => {
+    connected: (state, { payload }: PayloadAction<{ dataLayer: IDataLayer }>) => {
       console.log('[WS] Connected')
-      state.dataLayer = payload.dataLayer
+      dataLayerInterface = payload.dataLayer
       state.reconnectAttempts = 0
       state.error = undefined
     },
     error: (state, { payload }: PayloadAction<{ error: ErrorType }>) => {
       console.log('[WS] Error', payload.error)
       state.error = payload.error
-    }
+    },
+    save: () => {},
+    getInspectorPreferences: () => {},
+    setInspectorPreferences: (_state, _payload: PayloadAction<Partial<InspectorPreferences>>) => {},
+    getAssetCatalog: () => {},
+    undo: () => {},
+    redo: () => {},
+    importAsset: (_state, _payload: PayloadAction<ImportAssetRequest>) => {},
+    removeAsset: (_state, _payload: PayloadAction<Asset>) => {}
```

Comment:
are these above unsued?

Reply:
no, this generates the action with that payload type.
So then in the saga you can listen to this importAsset action